### PR TITLE
Add status queryparameter

### PR DESCRIPTION
**Pull request by angelbarrera92**
This PR adds the feature of filtering checkresults by status

### CODE REVIEW

**Comment by angelbarrera92** [comment]
Depends on: https://github.com/sighupio/fip-results-controller/pull/9

**Review by nandajavarma — COMMENTED** [review]
ignore my previous comment! Looks good!

**Review by nandajavarma — APPROVED** [review]

File: docs/releases/v0.4.0.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    "owner": "healthcheckcontroller",
+    "frequency": 1
+  },
+]
+```
+
+Optionally, pass the `status` query parameter to filter the `results`. Example:
+
+```bash
+$ curl http://localhost:8080/group/BookInfo/Product?limit=2&status=failed
```
Comment (nandajavarma) [inline docs/releases/v0.4.0.md:83]: I have beginners doubt. I see that in cronjobs the status is `Failed` or `Completed`(with first letter caps). Does the cronjob get the status in the same form? In which case would sending the `status=failed`(first letter small) fail?

Comment (angelbarrera92) [reply docs/releases/v0.4.0.md:83]: Ye... Currently it is case sensitive.

**Pull request merged**